helm-charts/splunk-otel-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: splunk-otel-collector
-version: 0.62.1
+version: 0.62.2
```

Comment:
if you don't mind - can we revert that change here and in rendered files to keep the diff a bit smaller?